packages/stems/src/assets/styles/colors.css

```diff
@@ -62,6 +62,7 @@ root {
   --accent-blue: #1ba1f1;
   --accent-purple: #8e51cf;
 
+  --special-green: #0f9348;
```

Comment:
i also thought this was a new color as i couldn't find any references to it.. but julian seems to think otherwise? 
https://www.figma.com/file/ClKssErI6tIMmZFwQifp9f?node-id=1423:68949&mode=dev#522669502

I named it `specialLightGreen1` in mobile, I think your name matches what's in figma tho. i'll change.